index.html

```diff
@@ -1,29 +1,116 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-  </head>
-  <body>
-    <header>
-      <div class="container"></div>
-    </header>
-
-    <main>
-      <section class="contact">
-        <div class="container"></div>
-      </section>
-    </main>
-
-    <footer>
-      <section class="cta">
-        <div class="container"></div>
-      </section>
-
-      <section class="copyright">
-        <div class="container"></div>
-      </section>
-    </footer>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <div class="container"></div>
+      <div>
+        <img src="assets/svg/logo.svg" />
+      </div>
+      <div><h2>WPH</h2></div>
+      <nav>
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">Products</a></li>
+          <li><a href="#">Resources</a></li>
+          <li><a href="#">Pricing</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="contact">
+        <div class="container">
+          <div class="contact-wrapper">
+            <div class="contact-header">
+              <p>Contact Us</p>
+              <h1>Get in touch</h1>

```

Comment:
Gunakan `<h2>` pada `<h1>Get in touch</h1>`. Ini bukan judul utama. Biasanya judul utama ada di hero section.